eclair-node/src/main/resources/application.conf

```diff
@@ -27,9 +27,10 @@ akka {
   cluster.roles = [backend]
   coordinated-shutdown.terminate-actor-system = on
   coordinated-shutdown.exit-jvm = on
-  //It is recommended to load the extension when the actor system is started by defining it in akka.extensions
-  //configuration property. Otherwise it will be activated when first used and then it takes a while for it to be populated.
-  extensions = ["akka.cluster.pubsub.DistributedPubSub"]
+  // It is recommended to load the extension when the actor system is started by defining it in akka.extensions
+  // configuration property. Otherwise it will be activated when first used and then it takes a while for it to be populated.
+  // Uncomment the line below in cluster mode
+  #extensions = ["akka.cluster.pubsub.DistributedPubSub"]
```

Comment:
nit: why a comment with `#` whereas we use `//` everywhere else?

Reply:
It was to separate between comment and disable feature, we seem to be using this convention in `reference.conf` (cf `backup-notify-script`, `override-features`...)

Reply:
Ok I thought so, makes sense.